docs/Adding-Software-to-Homebrew.md

```diff
@@ -186,6 +237,24 @@ Other commonly used stanzas are:
 
 Additional [`artifact` stanzas](Cask-Cookbook.md#at-least-one-artifact-stanza-is-also-required) may be needed for special use cases. Even more special-use stanzas are listed at [Optional Stanzas](Cask-Cookbook.md#optional-stanzas).
 
+#### Generating a token for the cask
+
+The cask **token** is the mnemonic string people will use to interact with the cask via `brew install`, etc. The name of the cask **file** is simply the token with the extension `.rb` appended.
```

Comment:
```suggestion
The cask **token** is the name people will use to interact with the cask via `brew install`, etc. The name of the cask **file** is simply the token with the extension `.rb` appended.
```

Reply:
Fixed in 1de9399aa7fb529f1387491ba1359ea71be5f831